docs/patterns/components/Alert/index.js

```diff
@@ -22,6 +22,8 @@ export default class Demo extends Component {
     return (
       <div>
         <h1>Alert</h1>
+        <h2>Component Description</h2>
+        <p>Shows a modal with a message. Optional warning styling.</p>
         <h2>Demo</h2>
```

Comment:
I noticed that this doc isn't using the `Demo` component. We could make an issue to update it.

Reply:
Issue #764 created!